pkgs/games/openxray/default.nix

```diff
@@ -49,20 +52,27 @@ stdenv.mkDerivation rec {
   cmakeBuildType = "RelWithDebInfo";
   dontStrip = true;
 
-  postInstall = ''
-    # needed because of SDL_LoadObject library loading code
+  # Because we work around https://github.com/OpenXRay/xray-16/issues/1224 by using GCC,
+  # we need a followup workaround for Darwin locale stuff when using GCC:
+  # runtime error: locale::facet::_S_create_c_locale name not valid
+  postInstall = lib.optionalString stdenv.hostPlatform.isDarwin ''
     wrapProgram $out/bin/xr_3da \
-      --prefix LD_LIBRARY_PATH : $out/lib
+      --run 'export LC_ALL=C'
   '';
 
+  # dlopens its own libraries, relies on rpath not having its prefix stripped
+  dontPatchELF = true;
+
+  passthru.updateScript = gitUpdater { };
+
   meta = with lib; {
-    mainProgram = "xray-16";
+    mainProgram = "xr_3da";
     description = "Improved version of the X-Ray Engine, the game engine used in the world-famous S.T.A.L.K.E.R. game series by GSC Game World";
     homepage = "https://github.com/OpenXRay/xray-16/";
     license = licenses.unfree // {
       url = "https://github.com/OpenXRay/xray-16/blob/${version}/License.txt";
     };
     maintainers = with maintainers; [ OPNA2608 ];
-    platforms = [ "x86_64-linux" "i686-linux" ];
+    platforms = [ "x86_64-linux" "i686-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
```

Comment:
How about just linux+darwin?

Reply:
Because for our usage x86_64 and aarch64 is basically linux because the other platforms are all so low in tier that they don't really mater. This field mostly used for hydra that it doesn't continuously build things that are known to not work.